readthedocs/core/signals.py

```diff
@@ -84,23 +82,15 @@ def decide_if_cors(sender, request, **kwargs):  # pylint: disable=unused-argumen
 
 @receiver(pre_delete, sender=settings.AUTH_USER_MODEL)
 def delete_projects_and_organizations(sender, instance, *args, **kwargs):
-    # Here we count the owner list from the projects that the user own
-    # Then exclude the projects where there are more than one owner
-    # Add annotate before filter
-    # https://github.com/rtfd/readthedocs.org/pull/4577
-    # https://docs.djangoproject.com/en/2.1/topics/db/aggregation/#order-of-annotate-and-filter-clauses # noqa
-    projects = (
-        Project.objects.annotate(num_users=Count('users')
-                                 ).filter(users=instance.id
-                                          ).exclude(num_users__gt=1)
-    )
+    user = instance
+    projects = get_projects_last_owner(user)
```

Comment:
Maybe "only" is better than "last" for this: `get_projects_only_owner(user)`. There are other places where we can change this as well. At least for me, is easier to read.